### PR TITLE
Docs: Update issue templates to ask for PRs

### PR DESCRIPTION
**Pull request by nzakas**
**What is the purpose of this pull request? (put an "X" next to item)**

[x] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/rule-change-proposal.md))
[ ] Add autofixing to a rule
[ ] Add a CLI option
[ ] Add something to the core
[ ] Other, please explain:

**What changes did you make? (Give an overview)**

Added a question to each issue template asking if the person creating the issue is willing to implement the change they are requesting. I thought at one point we had this, and I think it's a good way to nudge people into contributing rather than just expecting that someone else will implement the suggestion.

**Is there anything you'd like reviewers to focus on?**

Nothing in particular.

### CODE REVIEW

**Comment by not-an-aardvark** [comment]
Bikeshed: Could we reword this to something like "Would you be willing to submit a pull request" rather than "Will you submit a pull request"? It seems like the current wording might be interpreted as a request that someone work on a PR as soon as they create the issue, without waiting for feedback. People often report intended behavior as a bug if they misunderstand how it's supposed to work, and in those cases I wouldn't want them to preemptively spend their time on a PR only to be told later that the current behavior is correct.

**Comment by nzakas** [comment]
Fair point. I'd prefer to use "Are you willing..." instead of "Would you be willing..." as the latter is a bit more passive than I'm comfortable with. I'll make that change.
